services/common/src/components/forms/RenderSubmitButton.tsx

```diff
@@ -11,31 +11,36 @@ interface RenderSubmitButtonProps {
   disableOnClean?: boolean;
   iconButton?: boolean;
   icon?: ReactNode;
+  disabled?: boolean;
+  loading?: boolean;
 }
 
 const RenderSubmitButton: FC<RenderSubmitButtonProps> = ({
   buttonText = "Save Changes",
   buttonProps,
   disableOnClean = true,
   iconButton = false,
-  icon
+  icon,
+  ...props
 }) => {
   const { formName, isEditMode } = useContext(FormContext);
   const submitting = useSelector(isSubmitting(formName));
   const isFormDirty = useSelector(isDirty(formName));
-  const disabled = submitting || (!isFormDirty && disableOnClean);
+  const disabled = props.disabled || submitting || (!isFormDirty && disableOnClean);
+  const className = `${buttonProps?.className ?? ""} form-btn`;
```

Comment:
in order to accommodate some older files, added params for disabled & loading that will allow more control.